unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/BuilderMode/Context/Context.cs

```diff
@@ -80,15 +80,18 @@ public void Dispose()
         }
     }
 
-    public class EditorContext
+    public class EditorContext : IEditorContext
     {
 
         private const string GOD_MODE_DYNAMIC_VARIABLE_PATH = "ScriptableObjects/GodModeVariables";
         private const string FIRST_PERSON_DYNAMIC_VARIABLE_PATH = "ScriptableObjects/FirstPersonVariables";
 
         //Scriptable Objects
-        public BIWGodModeDynamicVariables godModeDynamicVariablesAsset { get; private set; }
-        public BIWFirstPersonDynamicVariables firstPersonDynamicVariablesAsset { get; private set; }
+        internal BIWGodModeDynamicVariables godModeDynamicVariablesAssetReference;
+        internal BIWFirstPersonDynamicVariables firstPersonDynamicVariablesAssetReference;
+
+        public BIWGodModeDynamicVariables godModeDynamicVariablesAsset => godModeDynamicVariablesAssetReference;
+        public BIWFirstPersonDynamicVariables firstPersonDynamicVariablesAsset => firstPersonDynamicVariablesAssetReference;
```

Comment:
Let's sort this a little bit. You can put all the interface members together (and even add the interface name as a prevfix: `IEditorContext.firstPersonDynamicVariableAsset`) for clarity.